docs/development.rst

```diff
@@ -94,10 +95,13 @@ Run the Tests
 #. Install test requirements::
 
     pip install -r requirements-test.txt
+    npm install -g gulp
```

Comment:
Thanks for the feedback. We could install Gulp just locally. Installing it globally allows it to be used as a command.